relay/api/exchange/resources.py

```diff
@@ -55,7 +61,20 @@ def get(self, args):
         }
 
 
+class OrderDetail(Resource):
+
+    def __init__(self, trustlines: TrustlinesRelay) -> None:
+        self.trustlines = trustlines
+
+    def get(self, order_hash: str):
+        order_orm = self.trustlines.orderbook.get_order_by_hash(bytes.fromhex(order_hash[2:]))
```

Comment:
You have to validate that the provided string is a valid hash. Right now just trying for exapme 1234 for the hash results in an internal server error: ValueError: non-hexadecimal number found in fromhex() arg at position 0